internal/node/node.go

```diff
@@ -130,3 +125,28 @@ func removeLabels(node *v1.Node, labels []string) {
 		)
 	}
 }
+
+func getLabelsToRemove(node *v1.Node, nmcObj *v1beta1.NodeModulesConfig) []string {
+	labelsToRemove := make([]string, 0)
```

Comment:
```suggestion
	var labelsToRemove []string
```

Can we make this instead? The difference is that with the current code and empty slice will be treated as a pointer to an empty array while the suggestion will return `nil` in this case which I think is preferable.

WDYT?

Reply:
Thought about it also, if we return `nil` its like we return nothing, but if we return an empty slice, its like we are saying we did not find any labels to remove.
IDM to switch it back though but that was my thought when I changed that.